lib/_stream_transform.js

```diff
@@ -90,7 +91,7 @@ function afterTransform(stream, er, data) {
 
   if (!cb) {
     return stream.emit('error',
-                       new Error('write callback called multiple times'));
+                       new errors.Error('ERR_MULTIPLE_CALLBACK'));
```

Comment:
Maybe to be consistent with my other suggestion:
'ERR_TRANSFORM_MULTIPLE_CALLBACK'.